### PR TITLE
Fixed some accessibility issues 

### PR DESCRIPTION
**Pull request by rishu2403**
## Context

Accessibility fixes for screwdriver.cd homepage

## Objective

Fixed some issues mentioned in 1.1.1 - Non- text content , 1.3.1 Info and relationships(headings) , 2.4.4 Link purpose from Accessibility report  https://github.com/screwdriver-cd/screwdriver/issues/2402#issue-850602234 

## References

https://www.w3.org/WAI/standards-guidelines/wcag/

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: .jekyll-cache/Jekyll/Cache/Jekyll--Converters--Markdown/e3/b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855 [hunk 1]
```diff
@@ -0,0 +1,2 @@
+I"
+:ET
```
Comment (tkyi) [inline .jekyll-cache/Jekyll/Cache/Jekyll--Converters--Markdown/e3/b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855:2]: should this file be checked in?

Comment (rishu2403) [reply .jekyll-cache/Jekyll/Cache/Jekyll--Converters--Markdown/e3/b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855:2]: I don't think so, these look like jekyll cache files.

Comment (tkyi) [reply .jekyll-cache/Jekyll/Cache/Jekyll--Converters--Markdown/e3/b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855:2]: hmm yes let's add it to the `_config.yml` under `exclude`: 
```
exclude:
  - .jekyll-cache
```
https://github.com/screwdriver-cd/homepage/blob/master/_config.yml#L23

Comment (rishu2403) [reply .jekyll-cache/Jekyll/Cache/Jekyll--Converters--Markdown/e3/b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855:2]: Thanks, done

**Review by tkyi — APPROVED** [review]

**Pull request merged**